DSLink.Tests/ResponderTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿using DSLink.Connection;
+using DSLink.Container;
+using DSLink.NET;
+using DSLink.Platform;
+using DSLink.Respond;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using PCLStorage;
+using Newtonsoft.Json.Linq;
+using DSLink.Nodes;
+using System.Threading.Tasks;
+using System;
+using DSLink.Nodes.Actions;
+
+namespace DSLink.Tests
+{
+    [TestFixture]
+    public class ResponderTests
+    {
+        private Configuration _config;
+        private DSLinkResponder _responder;
+        private Mock<IFolder> _mockFolder;
+        private Mock<AbstractContainer> _mockContainer;
+        private Mock<Connector> _mockConnector;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockFolder = new Mock<IFolder>();
+
+            BasePlatform.SetPlatform(new TestPlatform(_mockFolder));
```

Comment:
I think many of the SDKs tests are gonna use these setups. Might be good to think to extract a pre-setup Link and re-use it in other tests, and overwrite the configs that we explicitly test.